spec/controllers/state_file/questions/nj_household_rent_own_controller_spec.rb

```diff
@@ -69,16 +69,16 @@
         end
       end
 
-      context "when intake is both" do
-        let(:intake) { create :state_file_nj_intake, household_rent_own: "both" }
+      context "when intake is neither" do
```

Comment:
[dust] this could be an opportunity to reword these better than we had them -- `intake is neither` and `intake is both` are a bit vague... maybe `taxpayer neither owns nor rents` and `taxpayer both owns and rents`?